docs/guide/extending_page.md

```diff
@@ -106,6 +106,144 @@
             # ...
             visible: false
         ```
+    
+    #### Creating blocks with custom attributes
+    
+    You can create Page blocks with custom attributes.
+    The following steps demonstrate the process of creating a text custom attribute type.
```

Comment:
```suggestion
```
The line is not needed (what the section is about should be self-evident from the heading)